### DISCUSSION TITLE
Block parsing

### DISCUSSION THREAD

**Discussion by LidiaMil** [L-PAY/EVM_Parser#3] [Q&A] [answered]
Can we read any kind of data using block parsers?

**Comment by masxxiii** [comment, marked as answer]
The blockchain can be thought of as a big database that stores transactions. Every full node participating in the network has the same copy of the database. Overall, the blockchain is just a data structure for storing blocks. There is no encryption of the data stored in the blockchain because of which we can read and process the data present in the blockchain.
